### ISSUE TITLE
Color profile support?

### ISSUE THREAD

**Issue by kanyck** [easymodo/qimgv#89]
Hi!
Are there plans to add a color profile support for non-sRGB monitors?

**Comment by easymodo** [comment]
Yes but I haven't looked into it yet. So at least not for the next release.

**Comment by hooke007** [comment]
I hope it will support the icc/icm profile, which is definitely important for the wide color gamut monitor.

**Comment by novomesk** [comment]
Perhaps following resource would be useful:
https://littlecms.com/blog/2020/12/09/using-lcms2-on-qt/

**Comment by eddiezato** [comment]
This is definitely needed.

**Comment by Demez** [comment]
I would like to bump this, and bring up this image, which is an JPEG with an XYB ICC Color Profile, and without it, it's completely green

What it's supposed to look like:
![wollerau xyb__embeded_ICC_color](https://user-images.githubusercontent.com/30940677/233904019-396fc724-ed56-4440-8b46-a468bd0ac999.jpg)

How it looks in qimgv and other software lacking color profile support
![wollerau xyb__embeded_ICC_color2](https://user-images.githubusercontent.com/30940677/233904072-4f75927a-6272-4140-8d8b-f9fbc70e43cd.jpg)

**Comment by Kingproone** [comment]
https://commons.wikimedia.org/wiki/File:Defense.gov_News_Photo_080804-A-8725H-341.jpg

this image, when viewed in qimgv looks washed out
